src/scenes/Wallet/components/Balances/styles.js

```diff
@@ -0,0 +1,62 @@
+const styles = (theme) => ({
+  rootPaper: {
+    boxShadow: 'none',
+    borderRadius: theme.borderRadius,
+    background: theme.palette.background.paper,
+  },
+  containerGrid: {
+    padding: theme.padding.lg.px,
+    overflowX: 'hidden',
```

Comment:
actually we can try something like width: xxpx; overflowX: scroll so the ppl can see the full table if the window is too narrow.

Reply:
done. I used `minWidth` to avoid the table not reaching the full screen. also, I put this in the themes.

Reply:
awesome